src/Menu.java

```diff
@@ -72,25 +72,66 @@ public static void login() {
 		if(isValidUsernameAndPassword(username, password)) {
 			System.out.println("Logged in successfully.");
 			
-			//TODO: Create User object and display user menu
+			User.currentUser = User.getUserByUsername(username);
+			//TODO: Display user menu
 		} else {
 			System.out.println("Invalid Username or Password.");
 		}
-		
 	}
 	
 	/**
-	 * Checks if the username and password are greater than 5 characters in length
+	 * Checks if the username and password are valid
 	 * 
 	 * @param username string to be checked if valid 
 	 * @param password string to be checked if valid 
 	 * @return true if the username and password are valid, false otherwise
 	 */
 	public static Boolean isValidUsernameAndPassword(String username, String password) {
-		if(username.length() > 5 && password.length() > 5) {
-			return true;
-		} else {
-			return false;
+		return User.authentication(username, password);
+	}
+	
+	/**
+	 * User can Enter a valid username and password to create a new account
+	 */
+	public static void signup() {
```

Comment:
Can you break this up into smaller functions? It's just a little long.

Reply:
Also while you are fixing this, could you remove the TransactionTest.file and the test I made for the isValidUsername andPassword?